lib/rubocop/cop/sorbet/forbid_extend_t_sig_helpers_in_shims.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Sorbet
+      # This cop ensures RBI shims do not include a call to extend T::Sig
+      # or to extend T::Helpers
+      #
+      # @example
+      #
+      #   # bad
+      #   module SomeModule
+      #     extend T::Sig
+      #     extend T::Helpers
+      #
+      #     sig { returns(String) }
+      #     def foo; end
+      #   end
+      #
+      #   # good
+      #   module SomeModule
+      #     sig { returns(String) }
+      #     def foo; end
+      #   end
+      class ForbidExtendTSigHelpersInShims < RuboCop::Cop::Cop
+        include RangeHelp
+
+        MSG = 'Extending T::Sig or T::Helpers in a shim is unnecessary'
```

Comment:
Question: are there other things Sorbet allows you to extend that we want to avoid too? Like `T::Generic` maybe?

Reply:
We talked about T::Generic actually telling the user a lot about the class the shim is representing so we figured we'd let that fly. But T::Sig and T::Helpers don't really add anything to a shim.